src/utilities/dicomJson.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Converts a naturalized PersonName (json) to a part10 PersonName (string)
+ * @param {object} value json-style person name
+ * @returns {string} part10 dicom representation of PersonName
+ */
+function pnDenaturalize(value) {
+    if (value && typeof value === "object") {
+        return `${value.Alphabetic ?? ""}=${value.Ideographic ?? ""}=${
```

Comment:
It should only get the ideographic/phonetic separators if one of them exists.  That is:

if( value.Ideographic || value.Phonetic ) return your full string
return value.Alphabetic

That avoids you creating a string with the = in it when it isn't necessary (which changes the appearance in some systems).

Reply:
Notice the .replace just below which removes all end-of-string adjacent =. Thus if the full string is "ABC==" it becomes "ABC", "ABC=DEF=" becomes "ABC=DEF", and "ABC==GHI" stays the same.